scripts/package_firefox_extension.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+PREVIOUS_CURRENT_DIR=$(pwd)
+VERSION=$1
+cd ../tmp
+echo "Packaging unsigned Firefox extension, version $VERSION"
+zip -r ../build/kiwix-firefox-unsigned-extension-$VERSION.xpi www webextension manifest.json LICENSE-GPLv3.txt service-worker.js README.md
+# Install web-ext if it's not already installed
+if [ ! -f ../node_modules/web-ext/bin/web-ext ]; then
+    cd ..
+    npm install web-ext
+    cd tmp
```

Comment:
All those `cd` are difficult to follow.

If you assume that the script is run in the right directory you could write this part like this :
```
REVIOUS_CURRENT_DIR=$(pwd)
VERSION=$1

# Install web-ext if it's not already installed
if [ ! -f node_modules/web-ext/bin/web-ext ]; then
    npm install web-ext
endif

cd tmp
echo "Packaging unsigned Firefox extension, version $VERSION"
zip -r ../build/kiwix-firefox-unsigned-extension-$VERSION.xpi www webextension manifest.json LICENSE-GPLv3.txt service-worker.js README.md
[...]
```